.platform/hooks/postdeploy/db-migrate.sh

```diff
@@ -7,6 +7,7 @@ if [ -f /var/app/current/manage.py ]; then
 
     # Run migrations
     python manage.py migrate
+    echo `python manage.py showmigrations`
```

Comment:
Is this so it appears in the logs?

Reply:
yep
one of the previous deployments seemed to have everything but when i checked the migrations hadn't run at all (and there was no echo so I couldn't confirm that it actually hadn't run the command). I'm not sure if there's something of a disjoint between running the commands automatically and the state of the machine if I ssh in. Hopefully this will shed some light on things.